zetaclient/chains/evm/observer/inbound.go

```diff
@@ -470,11 +476,13 @@ func (ob *Observer) observeTSSReceive(ctx context.Context, startBlock, toBlock u
 }
 
 // checkAndVoteInboundTokenZeta checks and votes on the given inbound Zeta token
+// This is currently used for tracker votes only
```

Comment:
We should rename the function with `fromTracker` to make things clearer.

The isInternal argument would be confusing otherwise